showcase/src/main/kotlin/com/recco/showcase/main/PaletteSelection.kt

```diff
@@ -0,0 +1,300 @@
+                        .background(paletteColors.primary)
+                ) {
+                    Box(
+                        modifier = Modifier
+                            .fillMaxWidth(.5f)
+                            .fillMaxHeight(.5f)
+                            .align(Alignment.Center)
+                            .background(paletteColors.onPrimary)
+                    )
+                }
+                Box(
+                    modifier = Modifier
+                        .fillMaxWidth(.5f)
+                        .fillMaxHeight(.5f)
+                        .align(Alignment.TopEnd)
+                        .padding(start = .25.dp, bottom = .25.dp)
+                        .background(paletteColors.background)
+                ) {
+                    Box(
+                        modifier = Modifier
+                            .fillMaxWidth(.5f)
+                            .fillMaxHeight(.5f)
+                            .align(Alignment.Center)
+                            .background(paletteColors.onBackground)
+                    )
+                }
+                Box(
+                    modifier = Modifier
+                        .fillMaxWidth(.5f)
+                        .fillMaxHeight(.5f)
+                        .align(Alignment.BottomStart)
+                        .padding(end = .25.dp, top = .25.dp)
+                        .background(paletteColors.accent)
+                ) {
+                    Box(
+                        modifier = Modifier
+                            .fillMaxWidth(.5f)
+                            .fillMaxHeight(.5f)
+                            .align(Alignment.Center)
+                            .background(paletteColors.onAccent)
+                    )
+                }
+                Box(
+                    modifier = Modifier
+                        .fillMaxWidth(.5f)
+                        .fillMaxHeight(.5f)
+                        .align(Alignment.BottomEnd)
+                        .padding(start = .25.dp, top = .25.dp)
+                        .background(paletteColors.illustration)
+                )
+            }
+        }
+    }
+}
+
+@Preview
+@Composable
+private fun Preview() {
+    PaletteSelection(initiallyExpanded = true, selectionClickPalette = {})
+}
+
+@Composable
+private fun ReccoPalette.asTitle() = stringResource(
+    when (this) {
+        is ReccoPalette.Custom -> TODO()
+        ReccoPalette.Fresh -> R.string.fresh
+        ReccoPalette.Ocean -> R.string.ocean
+    }
+)
+
+@Composable
+private fun animateFloat(condition: Boolean, value: Float): Float {
+    val anim: Float by updateTransition(
+        targetState = condition,
+        label = ""
+    ).animateFloat(
+        transitionSpec = { tween(500) },
+        label = ""
+    ) { state ->
+        if (state) value else 0f
+    }
+
+    return anim
+}
+
+fun Modifier.noRippleClickable(onClick: () -> Unit): Modifier = composed {
```

Comment:
This extension function is already included in com.recco.internal.core.ui.extensions, maybe we want it to be imported from there ?

Reply:
I don't want to make a bad of example of consuming our own SDK xD Everything that it is inside an internal module, should not consumed by clients ;)